clients/context-registry/src/ContextRegistryClient.ts

```diff
@@ -6,8 +6,9 @@
  * @module ContextRegistry
  */
 import * as deepAssign from "deep-assign";
-import { assert } from "@bentley/bentleyjs-core";
-import { AuthorizedClientRequestContext, ECJsonTypeMap, RequestOptions, RequestQueryOptions, WsgClient, WsgInstance } from "@bentley/itwin-client";
+import { AuthorizedClientRequestContext, RequestOptions, RequestQueryOptions, } from "@bentley/itwin-client";
```

Comment:
Separate pr to remove all instances left of AuthorizedClientRequestContext?

Reply:
Yup, @kabentley has a PR that simplifies the request context concept and then @pbell97 has a PR that will simplify the auth workflow.  After both of those merge, my plan is to completely remove the `AuthorizedClientRequestContext` and have the token and ClientRequestContext as two separate concepts.